scipy-1.0/scipy-optimize.tex

```diff
@@ -73,59 +74,41 @@ \subsubsection*{Nonlinear Optimization}
-    point (i.e. a point $x^*$ for which $\nabla f(x^*) = 0$); this is \emph{not} a guarantee of convergence to a global minimum. The table also indicates which algorithms
-    can deal with constraints on the variables. We distinguish between: \textit{bound constraints} (i.e. $x^l \le x \le x^u$),
-    \textit{equality constraints} (i.e. $c_{\text{eq}}(x) = 0$) and \textit{inequality constraints} (i.e. $c_{\text{ineq}}(x) \ge 0$).}
-  \begin{tabular}{cccccccccccccc}
-      & \rotatebox{80}{\texttt{Nelder-Mead}} & \rotatebox{80}{\texttt{Powell}} & \rotatebox{80}{\texttt{COBYLA}} & \rotatebox{80}{\texttt{CG}} & \rotatebox{80}{\texttt{BFGS}}&  \rotatebox{80}{\texttt{L-BFGS-B}} & \rotatebox{80}{\texttt{SLSQP}} & \rotatebox{80}{\texttt{TNC}} & \rotatebox{80}{\texttt{Newton-CG}} & \rotatebox{80}{\texttt{dogleg}} & \rotatebox{80}{\texttt{trust-ncg}} & \rotatebox{80}{\texttt{trust-exact}} & \rotatebox{80}{\texttt{trust-krylov}} \\
-    \hline
-    Version added &  0.6* &  0.6* &  0.6* &  0.6* &  0.6* &  0.6* &  0.9 &  0.6* &  0.6* & 0.13 & 0.13 & 0.19 & 1.0 \\
-    \hline
-    Wrapper & & & \cmark & & & \cmark & \cmark & \cmark & &  & & & \cmark \\
+    has a superlinear rate for the parameters used in SciPy, but can achieve a quadratic convergence rate with other parameter choices.
+    These algorithms all guarantee convergence to a stationary
+    point (i.e. a point $x^*$ for which $\nabla f(x^*) = 0$); this is \emph{not} a guarantee of convergence to a global minimum. None of the listed algorithms 
+    can deal with constraints on the variables. A similar analysis for optimization algorithms added in older versions of SciPy is available in Table~\ref{tab:minimize-si}.}
+
+  \begin{tabular}{ccccc}
+      & \rotatebox{80}{\texttt{dogleg}} & \rotatebox{80}{\texttt{trust-ncg}} & \rotatebox{80}{\texttt{trust-exact}} & \rotatebox{80}{\texttt{trust-krylov}} \\
     \hline
-    \nth{1} derivatives &  & & & \cmark  & \cmark & \cmark & \cmark & \cmark & \cmark & \cmark & \cmark & \cmark & \cmark \\
+    Version added & 0.13 & 0.13 & 0.19 & 1.0 \\
     \hline
-    \nth{2} derivatives &  &  &  &  & $\sim$ & $\sim$ & $\sim$ & \cmark & \cmark & \cmark & \cmark & \cmark & \cmark \\
+    Wrapper & & & & \cmark \\
     \hline
     \makecell{Iterative Hessian \\
-    factorization} & & & &  & & & & \cmark & \cmark &  & \cmark &  & \cmark \\
-    \hline
-    Local convergence& & & & L & S &  L & S & S$^*$ & S$^*$ & Q & S$^*$ & Q & S$^*$  \\
+    factorization} &  & \cmark &  & \cmark \\
     \hline
-    Global convergence & & &  &   & \cmark & \cmark & \cmark & \cmark & \cmark & \cmark & \cmark & \cmark & \cmark  \\
+    Local convergence& Q & S$^*$ & Q & S$^*$  \\
     \hline
-    \makecell{Line-search (LS) or\\ trust-region (TR)} & Neither  & LS &  TR & LS & LS & LS & LS & LS & LS & TR & TR & TR & TR \\
+    \makecell{Line-search (LS) or\\ trust-region (TR)} & TR & TR & TR & TR \\
```

Comment:
All the same here, so could remove this row too--I'll wait for feedback on that though.